java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateJobs.java

```diff
@@ -119,6 +134,43 @@ public void createJobsForTable(TableProperties tableProperties) throws StateStor
         List<CompactionJob> compactionJobs = compactionStrategy.createCompactionJobs(activeFileInfosWithJobId, activeFileInfosWithNoJobId, allPartitions);
         LOGGER.info("Used {} to create {} compaction jobs for table {}", compactionStrategy.getClass().getSimpleName(), compactionJobs.size(), tableId);
 
+        if (forceCreateJobs) {
```

Comment:
It'd be nice if you could move this logic into its own method to make it a bit easier to read. It'd be nice to break this block up into smaller methods as well.

Reply:
Thanks, i've extracted this logic into a new method further below